src/client/src/Pages/CreateJob.tsx

```diff
@@ -1,7 +1,179 @@
 import React from 'react';
+import { Formik, Form, Field} from "formik";
+import Button from "@material-ui/core/Button";
+import * as Yup from 'yup'; 
 
-function CreateJob() {
-    return <h1>TODO: Adds components for the create jobs page</h1>;
+import FormikField from "../Components/JobForm/FormikField";
+import FormikSelect, { FormikSelectItem } from "../Components/JobForm/FormikSelect";
+
+
+interface FormValues {
+  title: string;
+  hours: string; 
+  major: string; 
+  targetYears: string; 
+  courses: string; 
+  startDate: string;
+  expirationDate: string; 
+  description: string; 
+  status: string; 
+  type: string; 
+  minSalary: string; 
```

Comment:
Is it possible to make all of these variables strings instead? Because in order for me to use YUP validation to ensure that the input is a numerical value, I need to accept a string value for both these variables.